habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -593,6 +615,13 @@ def import_tri_mesh(mesh_file: str) -> List[mn.trade.MeshData]:
     return mesh_data
 
 
+# global cache of Receptacles to reduce load time costs
+# maps receptacle name (not unique name) to a Receptacle instance (NOTE: parent_object_handle must be updated per-user-instance)
+_rec_cache: Dict[
+    str, Union[Receptacle, AABBReceptacle, TriangleMeshReceptacle]
```

Comment:
Thank you, alex! do we ensure the uniqueness of the name over all scenes and all receptacle?

Reply:
Great question. No guarantees here, unfortunately! It *should* be unique, but likely is not across all datasets and there is not any logic to validate this.

I believe these will be unique within HSSD currently.
Could write some logic to validate the dataset. :thinking: